src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -24,7 +24,7 @@ public async Task<Stream> DownloadPackageArtifact(string repoName, string buildI
             var downloadUrl = await GetDownloadArtifactUrl(repoName, buildId, artifactName);
             if (!string.IsNullOrEmpty(downloadUrl))
             {
-                downloadUrl = downloadUrl.Split("?")[0] + "?format="+ format + "& subPath=" + filePath;
+                downloadUrl = downloadUrl.Split("?")[0] + "?format="+ format + "&subPath=" + filePath;
```

Comment:
How did you end up finding this issue? Did our logging correctly show the issue?